main.go

```diff
@@ -385,6 +400,66 @@ func prepareConfig(parseOpts *cliOptions, conf *ffuf.Config) error {
 	return errs.ErrorOrNil()
 }
 
+func parseRawRequest(parseOpts *cliOptions, conf *ffuf.Config) error {
+	file, err := os.Open(parseOpts.request)
+	if err != nil {
+		return fmt.Errorf("could not open request file: %s", err)
+	}
+	defer file.Close()
+
+	r := bufio.NewReader(file)
+
+	s, err := r.ReadString('\n')
+	if err != nil {
+		return fmt.Errorf("could not read request: %s", err)
+	}
+	parts := strings.Split(s, " ")
+	if len(parts) < 3 {
+		return fmt.Errorf("malformed request supplied")
+	}
+	// Set the request Method
+	conf.Method = parts[0]
+
+	for {
+		line, err := r.ReadString('\n')
+		line = strings.TrimSpace(line)
+
+		if err != nil || line == "" {
+			break
+		}
+
+		p := strings.SplitN(line, ":", 2)
+		if len(p) != 2 {
+			continue
+		}
+
```

Comment:
A minor UX detail here: ffuf recalculates `Content-Length` - header based on the actual request size regardless if a `Content-Length` header is provided or not. To avoid confusion from the printed banner that shows the parsed `Content-Length` header, I think we should skip adding it in the first place.

Reply:
Hey, this should be fixed in the latest commits/